### PR TITLE
ADM-795:[backend]fix: fix bug that export pipeline csv can not show canceled status

### PR DESCRIPTION
**Pull request by yulongcai**
## Summary

fix bug that export pipeline csv can not show canceled status

## Before

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## After

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## Note

_Null_

### CODE REVIEW

File: backend/src/test/java/heartbeat/service/report/CSVFileGeneratorTest.java [hunk 1]
```diff
@@ -83,6 +83,32 @@ void shouldConvertPipelineDataToCsvGivenCommitInfoNotNull() throws IOException {
...
+			.MOCK_PIPELINE_CSV_DATA_WITH_PIPELINE_STATUS_IS_CANCELED();
+		csvFileGenerator.convertPipelineDataToCSV(pipelineCSVInfos, mockTimeStamp);
+		String fileName = CSVFileNameEnum.PIPELINE.getValue() + "-" + mockTimeStamp + ".csv";
+		File file = new File(fileName);
+
+		assertTrue(file.exists());
+
+		FileInputStream fileInputStream = new FileInputStream(file);
+		BufferedReader reader = new BufferedReader(new InputStreamReader(fileInputStream));
+		String headers = reader.readLine();
```
Comment (mikeyangyun) [inline backend/src/test/java/heartbeat/service/report/CSVFileGeneratorTest.java:99]: `given when then` pattern, maybe need a new line here

**Review by mikeyangyun — APPROVED** [review]

File: backend/src/test/java/heartbeat/service/report/CSVFileGeneratorTest.java (hunk 1 above)
Comment (yulongcai) [reply backend/src/test/java/heartbeat/service/report/CSVFileGeneratorTest.java:99]: done

File: backend/src/test/java/heartbeat/service/report/PipelineServiceTest.java [hunk 2]
```diff
@@ -58,18 +57,11 @@ public class PipelineServiceTest {
...
-	private String endTime;
+	public String startTime = "endTime";
 
-	@BeforeEach
-	public void setup() {
-		startTime = "startTime";
-		endTime = "endTime";
-		token = "token";
-	}
+	public String endTime = "token";
```
Comment (guzhongren) [inline backend/src/test/java/heartbeat/service/report/PipelineServiceTest.java:64]: give the variable a meaningful value.
make code readable
@yulongcai

**Review by guzhongren — COMMENTED** [review]
Change with the comments
@yulongcai

File: backend/src/test/java/heartbeat/service/report/PipelineServiceTest.java (hunk 2 above)
Comment (yulongcai) [reply backend/src/test/java/heartbeat/service/report/PipelineServiceTest.java:64]: done, pair with shiqi, on her pr

**Pull request merged**